### ISSUE TITLE
Update Accordion A11y Docs

### ISSUE THREAD

**Issue by jake-costa** [jpmorganchase/salt-ds#4814]
### Related page(s)

https://www.saltdesignsystem.com/salt/components/accordion/accessibility

### Kind of issue

Missing information

### Description

Adding Known Issue with Safari/VoiceOver - does not report state changes dynamically. (e.g. expanded/collapsed) Not a Salt Bug

Is a Safari/Mac/VoiceOver bug.

Add A11y best practices

### Browser

- [ ] Chrome
- [x] Safari
- [ ] Firefox
- [ ] Microsoft Edge

### Operating system

- [x] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### Are you a JPMorgan Chase & Co. employee?

- [x] I am an employee of JPMorgan Chase & Co.
